io.github.amourspirit.LibrePythonista_PyEditor.yml

```
@@ -0,0 +1,46 @@
+app-id: io.github.amourspirit.LibrePythonista_PyEditor
+runtime: org.gnome.Platform
+runtime-version: '47'
+sdk: org.gnome.Sdk
+command: cell_edit
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --filesystem=home
+  - --filesystem=~/.var/app/org.libreoffice.LibreOffice:ro
+
+modules:
+  - name: cell_edit
+    buildsystem: simple
+    build-commands:
+      - pip3 install --verbose --exists-action=i --no-index --find-links="file://${PWD}"
+        --prefix=${FLATPAK_DEST} "librepythonista_python_editor==0.1.8" --no-build-isolation
+      - install -Dm755 cell_edit /app/bin/cell_edit
+      - install -Dm644 icon.svg /app/share/icons/hicolor/scalable/apps/$FLATPAK_ID.svg
+      - install -Dm644 $FLATPAK_ID.metainfo.xml $FLATPAK_DEST/share/metainfo/$FLATPAK_ID.metainfo.xml
```

Same as above. The metainfo file is actually under `misc/` on your repo:

```suggestion
      - install -Dm644 misc/$FLATPAK_ID.metainfo.xml $FLATPAK_DEST/share/metainfo/$FLATPAK_ID.metainfo.xml
```